packages/fmg-nitro-adjudicator/contracts/ConsensusApp.sol

```diff
@@ -159,33 +159,34 @@ contract ConsensusApp {
     require(
       commitment.furtherVotesRequired == 0,
       "ConsensusApp: 'furtherVotesRequired' must be 0 during consensus."
-      ); 
+      );
     require(
       commitment.proposedAllocation.length == 0,
       "ConsensusApp: 'proposedAllocation' must be reset during consensus."
-      ); 
+      );
     require(
       commitment.proposedDestination.length == 0,
       "ConsensusApp: 'proposedDestination' must be reset during consensus."
-    ); 
-  } 
+    );
+  }
 
   function validateProposeCommitment(
     ConsensusCommitment.ConsensusCommitmentStruct memory commitment
   ) internal pure {
     require(
       commitment.furtherVotesRequired != 0,
       "ConsensusApp: 'furtherVotesRequired' must not be 0 during propose."
-      ); 
+      );
     require(
-      commitment.proposedAllocation.length > 0,
-      "ConsensusApp: 'proposedAllocation' must not be reset during propose."
-      ); 
+      commitment.proposedDestination.length > 0,
+      "ConsensusApp: 'proposedDestination' must not be reset during propose."
+      );
     require(
-      commitment.proposedDestination.length == commitment.proposedAllocation.length,
+      commitment.proposedAllocation.length == 0 || // in case it's a guarantor channel
```

Comment:
Should we update the require text to match the new condition?

Reply:
I think it's ok for now: In practice, if you're in a guarantor channel, `proposedAllocation` should always be `[]`, so you will never revert (which is the only case when the message is confusing).

Reply:
Actually, I just changed it.